plugins/dell-dock2/fu-dell-dock2-ec.c

```diff
@@ -0,0 +1,972 @@
+/*
+ * Copyright 2024 Dell Technologies
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR MIT
+ */
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-dell-dock2-common.h"
+
+#define BIT_SET(x, y)	(x |= (1 << y))
+#define BIT_CLEAR(x, y) (x &= (~(1 << y)))
```

Comment:
We've got a similar macro now in this plugin, dell-dock, and parade-usb-hub.  Would it make sense to maybe have such a macro moved to libfwupdplugin?

Reply:
I'm on it!

Reply:
I will remove `BIT_SET` and `BIT_CLEAR` from this plugin because K2 will do global lock/unlock.